scipy/spatial/distance.py

```diff
@@ -1398,6 +1398,10 @@ def pdist(X, metric='euclidean', *args, **kwargs):
         The inverse of the covariance matrix for Mahalanobis.
         Default: inv(cov(X.T)).T
 
+        out : ndarray.
+        The output array
```

Comment:
This needs to be indented. And nitpick: remove the ``.`` behind ``ndarray`` on the line above

Reply:
Removed the dots here and above. I think the indentation is right because it is part of an already indented list. I tried for the other PR (the lines above have the same problem), if you indent them you will get an error.

Reply:
You're right. Didn't see that in the diff view.